<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShaderMessageViewer</class>
 <widget class="QFrame" name="ShaderMessageViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>469</width>
    <height>303</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>X Shader Messages @ EID Y</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="toolbarHorizontalLayout">
      <property name="spacing">
        <number>5</number>
      </property>
      <property name="leftMargin">
        <number>0</number>
      </property>
      <property name="topMargin">
        <number>2</number>
      </property>
      <property name="rightMargin">
        <number>6</number>
      </property>
      <property name="bottomMargin">
        <number>2</number>
      </property>
      <item>
        <widget class="QToolButton" name="exportButton">
          <property name="text">
            <string>Export</string>
          </property>
          <property name="icon">
            <iconset resource="../Resources/resources.qrc">
              <normaloff>:/save.png</normaloff>:/save.png
            </iconset>
          </property>
          <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextBesideIcon</enum>
          </property>
          <property name="popupMode">
            <enum>QToolButton::MenuButtonPopup</enum>
          </property>
          <property name="autoRaise">
            <bool>true</bool>
          </property>
        </widget>
      </item>
      <item>
        <spacer name="toolbarHorizontalSpacer">
          <property name="orientation">
            <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
            <size>
              <width>40</width>
              <height>20</height>
            </size>
          </property>
        </spacer>
      </item>
    </layout>
   </item>
   <item>
    <widget class="RDLabel" name="label">
     <property name="text">
      <string>These are messages from @1234 - DrawIndexed(6, 1)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="RDLabel" name="staleStatus">
     <property name="text">
      <string>The results are out of date because of reasons.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="filterGroup">
     <property name="title">
      <string>Filter</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item>
       <widget class="QFrame" name="stageFilters">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="filterLabel">
           <property name="text">
            <string>Show messages from:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="vertex">
           <property name="text">
            <string>Vertex</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="hull">
           <property name="text">
            <string>Hull</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="domain">
           <property name="text">
            <string>Domain</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="geometry">
           <property name="text">
            <string>Geometry</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="pixel">
           <property name="text">
            <string>Pixel</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="textFilterFrame">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="RDLineEdit" name="filter">
           <property name="placeholderText">
            <string>Filter message text</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="filterButton">
           <property name="text">
            <string>Filter</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="RDTreeWidget" name="messages">
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ContiguousSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="showDropIndicator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="indentation">
      <number>0</number>
     </property>
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <property name="itemsExpandable">
      <bool>false</bool>
     </property>
     <property name="allColumnsShowFocus">
      <bool>true</bool>
     </property>
     <property name="expandsOnDoubleClick">
      <bool>false</bool>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <attribute name="headerStretchLastSection">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RDTreeWidget</class>
   <extends>QTreeView</extends>
   <header>Widgets/Extended/RDTreeWidget.h</header>
  </customwidget>
  <customwidget>
   <class>RDLabel</class>
   <extends>QLabel</extends>
   <header>Widgets/Extended/RDLabel.h</header>
  </customwidget>
  <customwidget>
   <class>RDLineEdit</class>
   <extends>QLineEdit</extends>
   <header>Widgets/Extended/RDLineEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
